pep-0581.rst

```diff
@@ -69,29 +74,33 @@ not currently available on Roundup / bpo.
   systematic filtering of emails.
 
 - Additional privacy, such as offering the user a choice to hide an
-  email address, while still allowing communication with the user through @-mentions.
+  email address, while still allowing communication with the user
+  through @-mentions.
+
 
 Issues with Roundup / bpo
 -------------------------
 
 - Less than five people maintain bpo. Some of them are core developers.
 
-- It is in Mercurial. Without any CI available, it puts heavy burden on the few
-  existing maintainers in terms of reviewing, testing, and applying patches.
+- The upstream Roundup code is in Mercurial. Without any CI available,
+  it puts heavy burden on the few existing maintainers in terms of
+  reviewing, testing, and applying patches.
 
-- At its current state, it is not equipped to accept lots of contributions from
-  people who aren't already familiar with the code base.
+  There is an open discussion about moving the source code of bpo to
+  GitHub [#]_. If the source code of bpo does move to GitHub, it will
+  become difficult to update patches from upstream. But as long as it
+  is in Mercurial, it is difficult to maintain and onboard new
+  contributors.
 
-- The upstream Roundup is in Mercurial. There is an open discussion about
-  moving the source code of bpo to GitHub [#]_. If the source code of
-  bpo does move to GitHub, it will become difficult to update patches from
-  upstream. But as long as it is in Mercurial, it is difficult to maintain
-  and onboard new contributors.
+- In its current state, the project is not equipped to accept lots of
+  contributions from people who aren't already familiar with the code
+  base.
 
 - The user interface needs an update and redesign. It will require UX/UI research
   to keep it up to date with current web standards, including accessibility.
 
-- Email address is exposed. There is no choice to mask it.
+- Users email addresses are exposed. There is no option to mask it.
```

Comment:
Roundup can be modified to mask usernames and/or email addresses. See https://issues.roundup-tracker.org/issue2551034 for example: Usernames masked as `[hidden]` there.